stack.yaml

```diff
@@ -1,4 +1,4 @@
-resolver: lts-12.18
+resolver: lts-13.0
```

Comment:
The `transform` combinator requires `-XQuantifiedConstraints`. We can specialize write `shortCircuit` and `stateful` directly in older versions of GHC to avoid this.